src/cheri_sys_exceptions.sail

```diff
@@ -9,31 +9,16 @@ type ext_exception = cheri_cause
 function handle_trap_extension(p : Privilege, pc : xlenbits, ccause : option(cheri_cause)) -> unit = {
```

Comment:
Can we now axe cheri_cause entirely and replace it with `unit` like in the base RISC-V model?